src/app/(tabs)/friends.tsx

```diff
@@ -1,9 +1,24 @@
-import { Text, View } from "@/src/components/Themed";
+import FriendList from "@/src/components/FriendList";
+import FriendRequestList from "@/src/components/FriendRequestList";
+import FriendSearchBar from "@/src/components/FriendSearchBar";
+import HabitInviteList from "@/src/components/HabitInviteList";
+import { Text, ScrollView } from "@/src/components/Themed";
 
 export default function FriendsTab() {
   return (
-    <View className="flex-1 p-4">
-      <Text>Content</Text>
-    </View>
+    <ScrollView className="flex-1 p-4"
+      contentContainerStyle={{ paddingBottom: 100 }}
+      >
+      <Text className="text-2xl font-bold">New friend requests</Text>
+      <FriendRequestList />
+
+      <Text className="text-2xl font-bold">New habit invite</Text>
+      <HabitInviteList />
+
+      <Text className="text-2xl font-bold">My friends</Text>
+      <FriendSearchBar />
+      <FriendList />
```

Comment:
Everything is too close together; follow design and put more of a gap between sections. Also, title text should be `text-xl` so that it is 20px, matching the design.